src/components/LandingPage.tsx

```diff
@@ -0,0 +1,25 @@
+import Link from 'next/link'
+import styles from '@styles/landingPage.module.css'
+import dentalScheduling from '@public/dentalScheduling.png'
+import Image from 'next/image'
+
+const LandingPage = () => {
+  return (
+    <div className={styles.container}>
+      <Image
+        src={dentalScheduling}
+        alt="image of calendar and fake dentures"
+        className={styles.img}
+      />
+      <h1 className={styles.header}>Welcome to Essential Dental Solutions</h1>
+      <p>This is an online appointment request system.</p>
+      <p>
+        If you are ready, click on &quot;Get Started&quot; below to proceed.
+      </p>
+      <Link href="/appointments">
```

Comment:
Good job @NuriyaAkh 🚀  There are a few suggestions:
1. In HTML, links and buttons are different elements, which serve different purposes:
- Links take the user to a new location, such as a new web page or new section of the current page.
- Buttons trigger some action, such as showing content on the page that was previously hidden, playing a video, or submitting a form.

In the above use case, the intent is to route the user to different page or route. So, since you have used `<Link>` that makes sense, but I don't think, you need to use a `<button>` element for it. You can style a link to look differently as you want. Also, we should never nest a button and links together. [Is It Possible to Nest an HTML <button> Element Inside an <a> Element in HTML5](https://www.w3docs.com/snippets/css/is-it-possible-to-nest-an-html-button-element-inside-an-a-element-in-html5.html)

2. I believe we need to look at where the **Get Started** button should route to. If it should route a user to book an appointment, since Amanda has worked on creating an appointment form, we should take it as a future issue to create a route for the new appointment form and then, use that route here.

Reply:
Thank you for your valuable feedback and suggestions, @5hraddha I really appreciate your input. Based on your advice, I have removed the <button> element and instead created a new appointment route for the "Get Started" button.
 If you have any further suggestions or ideas, please feel free to share them.